changelog/v1.19.0-beta14/aws-lambda-429-fix.yaml

```diff
@@ -0,0 +1,20 @@
+changelog:
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: solo-io
+    dependencyRepo: envoy-gloo
+    dependencyTag: v1.33.0-patch2
+    issueLink: https://github.com/solo-io/solo-projects/issues/7787
+  - type: FIX
+    issueLink: https://github.com/kgateway-dev/kgateway/issues/10192
+    resolvesIssue: true
+    description: >-
+      Previously, Gloo did not provide any means to determine whether a request
```

Comment:
Should we have an e2e/integtation test that verifies this functionality? We have a set of aws tests that run in memory here: https://github.com/solo-io/gloo/blob/main/test/e2e/aws_test.go, so I figure that would be the place to update, if we thought it was valuable

Reply:
i was thinking about it. in my opinion, it's already tested very cleanly in `envoy-gloo`: https://github.com/solo-io/envoy-gloo/pull/407/files#diff-b04e01bfc16426121d645e12d2a61f2a12984d8dc13187ddb6538d714b0ee963R53-R69

so i don't think it would add a whole lot of value, but happy to add it if you think it would be a good idea. it's always possible that it would have some sort of benefit i haven't thought of...